packages/models/scripts/seed.ts

```diff
@@ -226,6 +229,33 @@ async function main(): Promise<void> {
   const tutorManager = first(addedTutorManagers);
   if (!tutorManager) throw new Error("TutorManager not found; should never happen.");
 
+  // seeding ratings data
+  const addedRatings: IRating.Self[] = await knex.transaction(async () => {
+    return await Promise.all(
+      students.map(async (student) => {
```

Comment:
This will make a student rate one tutor. Let's increase the rating count by making each student rates all tutors.